src/hooks/useOracleReader.ts

```diff
@@ -39,6 +39,7 @@ export default function useOracleReader(state: oracle.types.state.State) {
   const disputer = request?.disputer;
   const proposer = request?.proposer;
   const chainId: ChainId = Number(state.inputs?.request?.chainId || 1);
+  const provider = read.provider(chainId);
```

Comment:
you had mentioned this can sometimes throw an error if we are on an unsupported chain. we might need to catch that (would probably crash app) and set provider to undefined and check that in the gov rules

Reply:
The update fixes the issue that was crashing the testnet deployment. I am still seeing `ExistenceError: Provider not found on chainid: 1` on the testnet deployment but couldn't that be caused by the chainId fallback being set to 1 and mainnet isn't enabled?

Reply:
hmm yea thats a good point. im open to other suggestions here, i think easiest is to try catch this, but u could try to change the chainid logic